code/datums/map_elements.dm

```diff
@@ -23,12 +23,8 @@ var/list/datum/map_element/map_elements = list()
 	if(!location && objects.len)
 		location = locate(/turf) in objects
 
-	//Build powernets
-	for(var/obj/structure/cable/C in objects)
```

Comment:
It got moved to the `spawned_by_map_element` proc in cable.dm, which is called in the new lines here.